tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1956,6 +1956,26 @@ def isDirectToVgprDoable(state, tc):
       reject(state, "DirectToVgpr%c does not supports Sparse"%(tc))
       return False
 
+    # for DTVA, does not work with PGR0
+    if tc == 'A' and state["PrefetchGlobalRead"] == 0:
+      reject(state, "DirectToVgpr%c does not supports PrefetchGlobalRead == 0."%(tc))
+      return False
+    
+    # for DTVA, does not work with NN and TLDS0
+    if tc == 'A' and state["TransposeLDS"] == 0 and (not state["ProblemType"]["TransposeA"] and not state["ProblemType"]["TransposeB"]):
```

Comment:
Why tcA is relative with TransposeB?

Reply:
@hcman2 , this is also our question. Two tickets in this PR have similar issues. Such as:
The second one is DTVA1 + NN + TLDS=0 (But NT is good) depends on our test, and:
The third one  is DTVA1 + TT + Tail failed (But TN is good...) and DTVB1 + NN + Tail failed (But TN is good)